lib/embedded_ansible.rb

```diff
@@ -11,13 +17,114 @@ def self.enabled?
   end
 
   def self.running?
-    # TODO
-    false
+    services.map { |service| LinuxAdmin::Service.new(service).running? }.inject(:&)
+  end
+
+  def self.configure
+    configure_secret_key
+    with_inventory_file do |inventory_file_path|
+      setup_params = {
+        :e => "minimum_var_space=0",
+        :k => "packages,migrations,supervisor",
+        :i => inventory_file_path
+      }
+      AwesomeSpawn.run!(SETUP_SCRIPT, :params => setup_params)
+    end
+    stop
+  end
+
+  def self.start
+    with_inventory_file do |inventory_file_path|
+      setup_params = {
+        :e => "minimum_var_space=0",
+        :k => "packages,migrations",
+        :i => inventory_file_path
+      }
+      AwesomeSpawn.run!(SETUP_SCRIPT, :params => setup_params)
+    end
+  end
+
+  def self.stop
```

Comment:
Does it make sense to name this method "disable"?  I think we can assume disabling a service should  stop it but I don't think you can assume stopping a service should disable it.